### PR TITLE
Don't repeat flag in comment if already in disasm

### PR DESCRIPTION
**Pull request by kazarmy**

### CODE REVIEW

File: libr/core/disasm.c [hunk 1]
```diff
@@ -2617,8 +2617,12 @@ static void ds_print_ptr(RDisasmState *ds, int len, int idx) {
 					}
 				}
 				ALIGN;
-				ds_comment (ds, true, "%s; 0x%" PFMT64x "%s%s%s", esc, p,
-						*flag ? " ; " : "", flag, nl);
+				if (*flag && strstr(ds->opstr, flag)) {
```
Comment (radare) [inline libr/core/disasm.c]: missing space

**Comment by radare** [comment]
there's a segfault in nopskip 
screen shot 2017-03-25 at 03 02 38

File: libr/core/disasm.c (hunk 1 above)
Comment (kazarmy) [inline libr/core/disasm.c]: ok done

**Comment by kazarmy** [comment]
Should be ok now.

**Comment by radare** [comment]
Do we have tests for this?

**Comment by Maijin** [comment]
https://github.com/radare/radare2-regressions

**Comment by kazarmy** [comment]
Ok done.

**Comment by kazarmy** [comment]
I'll have a look at the agf segfault if nobody gets there first. Would be good to remove the ds->opstr check.

**Pull request merged**